hosting/libs/firebase/config.ts

```diff
@@ -0,0 +1,17 @@
+import { getAuth } from "firebase/auth";
+import { initializeApp, getApps } from "firebase/app";
+
+// Load .env variables
+const firebaseConfig = {
+  apiKey: process.env.NEXT_PUBLIC_FIREBASE_API_KEY,
+  authDomain: process.env.NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN,
+  projectId: process.env.NEXT_PUBLIC_FIREBASE_PROJECT_ID,
+  storageBucket: process.env.NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET,
+  messagingSenderId: process.env.NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID,
+  appId: process.env.NEXT_PUBLIC_FIREBASE_APP_ID,
+};
+
+const firebaseApp =
```

Comment:
We can optimizing this line code
```suggestion
const firebaseApp = initializeApp(firebaseConfig)
```